### ISSUE TITLE
Configuration not found after upgrade to 2.5.0-rc1

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/desktop#766]
### Steps to reproduce
1. Install 2.3.3
2. Configure accounts
3. Install 2.5.0-rc1
4. No accounts configured

On 2.3.3 the config file was stored in AppData\Local\Nextcloud while 2.5.0 will look for a config file in AppData\Roaming\Nextcloud.

@rullzer @camilasan

**Comment by klatka** [comment]
Is it possible to use the old config path?

The problem i have:
I use more than one device with the nextcloud sync client for windows. I want to have other folders synced for each device. Unfortunately this is not possible because the config file is saved in the roaming profile of the user i use for every device...

Sorry if my question does not belong in here

**Comment by Iolaum** [comment]
I think I 've run in this bug on my Fedora MATE 30 laptop. I had 2.5.2, uninstalled and downgraded to 2.3.3 and had to re-set up account. Then when I upgraded to 2.5.2 because I forgot to pin the package client used my old setup from when 2.5.2 was initially installed. I removed sync connection and downgraded to 2.3.3 waiting for issues like #876 to get fixed. When sync in 2.5.x+ gets reliable having to re-sync wouldn't be much of a problem.
